src/styles/App.css

```diff
@@ -1,13 +1,5 @@
 
 body {
-  background-color: aliceblue;
-}
-
-.book {
-  border: black solid;
-}
-
-.active-link {
-  color: orangered;
-  text-decoration: underline;
+  background-color: rgb(239, 239, 240);
+  margin: 0;
 }
```

Comment:
- [ ] Please, add a proper font-family to the project, this will make your style better. 🐶 
![image](https://user-images.githubusercontent.com/20563252/135683991-e6d379ff-9982-47ee-ae60-892c10db9718.png)

Reply:
What font family would you recommend?